frontend/src/pages/WorkspaceLocked/WorkspaceLocked.tsx

```diff
@@ -187,8 +187,26 @@ export default function WorkspaceBlocked(): JSX.Element {
 			children: (
 				<Row gutter={[24, 16]} justify="center">
 					{/* #FIXME: please suggest if there is any better way to loop in different columns to get the masonry layout */}
-					<Col span={10}>{renderCustomerStories((index) => index % 2 === 0)}</Col>
-					<Col span={10}>{renderCustomerStories((index) => index % 2 !== 0)}</Col>
+					<Col
+						span={10}
+						style={{
+							display: 'flex',
+							flexDirection: 'column',
```

Comment:
can you move these to css files and add a classname here. we should not add inline styles

Reply:
done